README.md

```diff
@@ -57,14 +57,16 @@ fcrepo.authHost=fcrepo
 reindexing.rest.host=0.0.0.0
 ```
 
-Then to start the Camel Toolbox, Fedora, Fuseki, and Solr containers run
+Then to start,  the Camel Toolbox, Fedora, Fuseki, and Solr containers run
 ```
+cd ./docker-compose
 docker-compose up -d
 ```
 
 If you need to rebuild the docker image, it can be done through docker compose as long as the `build` is specified
```

Comment:
I wonder if this is really needed since it's only saying run `docker-compose build`, which isn't that useful without providing an example of the changes needed for the compose file. Maybe we should have this more as an aside and link to the compose file documentation instead?